packages/react/react/src/utils/convertNewlineToJSX.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { convertNewlineToJSX } from './convertNewlineToJSX';
+
+describe('convertNewlineToJSX', () => {
+  it('converts newline characters into <br /> JSX elements', () => {
+    const input = 'There was a problem trying to sign in.\nplease try again.';
+    const output = [
+      <React.Fragment key={0}>There was a problem trying to sign in.</React.Fragment>,
```

Comment:
If Fragment is replaced with <></> in the `convertNewlineToJSX` code, this test will be broken, is that okay?

Reply:
It's my mistake.